src/tigerbeetle/main.zig

```diff
@@ -184,13 +184,22 @@ const Command = struct {
         const nonce = std.crypto.random.int(u128);
         assert(nonce != 0); // Broken CSPRNG is the likeliest explanation for zero.
 
+        // TODO Where should this be set?
+        const release_client_min = config.process.release;
+        const releases_bundled = &[_]vsr.Release{config.process.release};
+
+        log_main.info("version={}", .{constants.semver});
+        log_main.info("release={}", .{config.process.release});
```

Comment:
Do we need to print release? It is the tripple part of version, so maybe just print version until the two do diverge in practice? 

Alternatively, don't print version and print only release and git_commit.

I agree that we should print crucial info early, but I'd also love us to be as succinct as possible otherwise, as redundant and useless info in logs generally makes debugging harder.

Reply:
Good point! I kept `release` but removed `version`. (Reasoning: The other numbers we are printing are also releases. And the version's git commit is already covered a few lines down.)